packages/organizations/origin-organization-irec-api/migrations/1626682324674-UserName.ts

```diff
@@ -0,0 +1,15 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class UserName1626682324674 implements MigrationInterface {
+    name = 'UserName1626682324674';
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(
+            `ALTER TABLE "irec_connection" ADD "userName" character varying NOT NULL`
```

Comment:
This will fail on real database because it's NOT NULL

Reply:
Yep, it was never used.